app/javascript/controllers/media_player_controller.js

```diff
@@ -20,6 +20,10 @@ export default class extends Controller {
         const event = new CustomEvent('time-update', { detail: timestamp })
         window.dispatchEvent(event)
       })
+
+      // patch for https://github.com/sul-dlss/sul-embed/issues/2258
+      // TODO: check we can remove after videojs is updated to 8.19.2.
```

Comment:
Can we upgrade to 8.20.0 now and then drop this? https://github.com/videojs/video.js/commit/62f38446a5f01e85739bdb7207c7e992b57c1c86

Reply:
I should update the todo. The upgrade did not fix the issue. I am not quite sure what is going on but it seems to be specific to our sul-embed instance.